src/index.css

```diff
@@ -0,0 +1,342 @@
+ #breedsPage {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#breedsPage > *,
+#breedPage > *,
+#subBreedPage > * {
+  width: 80%;
+  margin: 1.5rem auto;
+}
+
+.pageTitle {
+  font-size: 2.5rem;
+  text-align: center;
+}
+
+.breedsIntro {
+  text-align: justify;
+  text-justify: distribute;
+  text-indent: 2rem;
+  font-size: 1.5rem;
+}
+
+.pagination-cta {
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+/*Pagination Container*/
+.breedsPaginationContainer, .subBreedsPaginationContainer {
+  display: flex;
+  flex-flow: row wrap;
+  list-style: none;
+  justify-content: center;
+}
+
+.paginationLink,
+.break > [role="button"] {
+  display:block;
+  padding: 1rem;
+  cursor:grab;
+}
+
+.activeLetter {
+  background-color: rgba(191, 151, 228,.4);
+}
+
+.nextLink, .prevLink {
+  display: block;
+  padding: 1rem;
+  cursor:grab;
+}
+/*End of Pagination Container classes*/
+
+/*Breeds*/
+#breeds {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.breedContainer {
+  display: flex;
+  justify-content: center;
+  flex: 33%;
+  min-width: auto;
+  margin: 20px 0;
+}
+
+.breedContainer a {
+  display: block;
+  width: 100%;
+  text-decoration: none;
+  font-weight: bold;
+  color: black;
+}
+
+.breedContainer .dogFigure {
+  margin: 0;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+}
+
+.dogFigure:hover {
+  animation: dogTime 1s ease-in-out 1s;
+}
+
+@keyframes dogTime {
+  from { }
+  to {}
+  /*0%{}
+  20%{}
+  40%{}
+  60%{}
+  80%{}
+  100%{}*/
+}
+
+.breedContainer .dogImage{
+  max-height: 40vh;
+  width: 100%;
+  object-fit: cover;
+  aspect-ratio: 1/1;
+}
+
+#breed, #subBreed {
+  display:grid;
+  grid: repeat(5,auto) / repeat(10,10%);
+}
+
+#subBreed .dogImage {
+  width: 100%;
+}
+
+.dogFigure > figCaption{
+  display: flex;
+  justify-content: center;
+  padding:0;
+}
+
+#breed .dogFigure, #subBreed .dogFigure {
+  margin:0;
+  display:flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#breed .dogFigure:nth-child(1),#subBreed .dogFigure:nth-child(1) { grid-area: 1 / 1 / 2 / 5; }
+#breed .dogFigure:nth-child(2),#subBreed .dogFigure:nth-child(2) { grid-area: 1 / 5 / 2 / 10;}
+#breed .dogFigure:nth-child(3),#subBreed .dogFigure:nth-child(3) { grid-area: 2 / 1 / 3 / 10; }
+#breed .dogFigure:nth-child(4),#subBreed .dogFigure:nth-child(4) { grid-area: 3 / 1 / 4 / 5; }
+#breed .dogFigure:nth-child(5),#subBreed .dogFigure:nth-child(5) { grid-area: 3 / 5 / 4 / 10;}
+#breed .dogFigure:nth-child(6),#subBreed .dogFigure:nth-child(6) { grid-area: 4 / 1 / 5 / 6; }
+#breed .dogFigure:nth-child(7),#subBreed .dogFigure:nth-child(7) { grid-area: 4 / 6 / 5 / 10; }
+#breed .dogFigure:nth-child(8),#subBreed .dogFigure:nth-child(8) { grid-area: 5 / 1 / 6 / 4; }
+#breed .dogFigure:nth-child(9),#subBreed .dogFigure:nth-child(9) { grid-area: 5 / 4 / 6 / 7; }
+#breed .dogFigure:nth-child(10),#subBreed .dogFigure:nth-child(10){grid-area: 5 / 7 / 6 / 10; }
```

Comment:
Could this be usable for more child elements than 10?

Reply:
With CSS only no and it's a pity. This layout would be sweet on the homepage.
With JS maybe It could work, although I haven't done it.
I wasted a lot of hours to find a way to make it look the same for any given number of elements with CSS.
The problem is that the grid-area is not accepting n-containing values.

However, the layout works smoothly for less than 10 children.